server/app/models/Application.java

```diff
@@ -117,4 +119,25 @@ public Application setSubmitTimeToNow() {
     this.submitTime = Instant.now();
     return this;
   }
+
+  /**
+   * Returns the latest application status value associated with the application.
+   *
+   * <p>This value is updated by DB triggers defined in conf/evolutions/default/44.sql which set the
+   * status to the latest ApplicationEventDetails event for the application with a type of
+   * "status_change". Attempts to update the status manually will be overridden by the trigger (and
+   * has associated tests confirming this).
+   */
+  public Optional<String> getLatestStatus() {
```

Comment:
Is there a reason to not make the full StatusEvent available here?  EG we can't as easily see the value for sendEmail or subsequent data we add.